Model/Indexer/AbstractIndexer.php

```diff
@@ -116,15 +116,76 @@ public function __construct(
      */
     abstract public function getIndexerId(): string ;
 
+    /**
+     * @inheritdoc
+     * @throws Exception
+     */
+    public function executeFull()
+    {
+        $indexerId = $this->getIndexerId();
+        $message = sprintf(
+            'Begin a full reindex for indexer "%s"',
+            $indexerId
+        );
+        $this->nostoLogger->info($message);
+        $this->doWork();
+        $this->nostoLogger->info("Finished full reindex");
+    }
+
+    /**
+     * @inheritdoc
+     * @throws Exception
+     */
+    public function executeList(array $ids)
+    {
+        $indexerId = $this->getIndexerId();
+        $idCount = count($ids);
+        $message = sprintf(
```

Comment:
Let's add here the total number of lines in cl table as well.

Reply:
It shouldn't be here. I'll explain.